test/publishing_api_v2_test.rb

```diff
@@ -11,7 +11,8 @@ def content_item_for_content_id(content_id, attrs = {})
       "content_id" => content_id,
       "title" => "Instructions for crawler robots",
       "description" => "robots.txt provides rules for which parts of GOV.UK are permitted to be crawled by different bots.",
-      "format" => "special_route",
+      "schema_name" => "special_route",
+      "document_type" => "special_route",
```

Comment:
Perhaps it would be good to use a different thing for `document_type` & `schema_name` to indicate that there's a difference (you can make something up for `document_type`).